hatasmota/discovery.py

```diff
@@ -392,6 +395,21 @@ def get_fan_entities(
     return fan_entities
 
 
+def get_camera_entities(
+    discovery_msg: dict,
+) -> list[tuple[TasmotaCameraConfig | None, DiscoveryHashType]]:
+    """Generate camera configuration."""
+    camera_entities: list[tuple[TasmotaCameraConfig | None, DiscoveryHashType]] = []
+
+    entity = None
+    discovery_hash = (discovery_msg[CONF_MAC], "cam", "cam", 0)
+    if CONF_CAM in discovery_msg and discovery_msg[CONF_CAM]:
+        entity = TasmotaCameraConfig.from_discovery_message(discovery_msg, "camera")
+    camera_entities.append((entity, discovery_hash))
+
+    return camera_entities
+
+
```

Comment:
Please move this to before `get_cover_entities` to keep the functions mostly sorted alphatebetically

Reply:
Done.